### ISSUE TITLE
Unable to install Spinnaker Version 1.29.2 Package on Ubuntu 22.04 LTS

### ISSUE THREAD

**Issue by SeungHyeonShin** [spinnaker/spinnaker#6795]
### Issue Summary:
Error installing Spinaker 1.29.2 package on Ubuntu 22.04.

### Cloud Provider(s):
N/A

### Environment:
- Ubuntu 22.04 LTS running in EC2.
- Halyard v1.54.0

### Feature Area:

Rosco

### Description:
*Expected Behaviour:*
Installing the spinnaker package using apt should succeed without any errors.

*Actual Behaviour:*
Package installation fails with error log below 'Package spinnaker-rosco is not available'.

```bash
Package spinnaker-rosco is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Version '1.12.2' for 'spinnaker-rosco' was not found
! ERROR Error encountered running script. See above output for more
  details.
```

In my opinion, this error appears to have occurred because the 1.12.2 version of spinnaker-rosco was not uploaded to the repository.

- apt repository : [https://us-apt.pkg.dev/projects/spinnaker-community/dists/apt/main/binary-all/Packages](https://us-apt.pkg.dev/projects/spinnaker-community/dists/apt/main/binary-all/Packages)

```bash
> apt-cache show spinnaker-rosco
Package: spinnaker-rosco
Source: spinnaker-rosco
Version: 1.12.1
Installed-Size: 119561
Maintainer: runner
Architecture: all
Depends: unzip
Description: spinnaker-rosco
Description-md5: 09be5d33c1f7d5ca72f2395dc00412f5
Filename: pool/apt/spinnaker-rosco_1.12.1_all_63871f6536369f1b084d126ee652ab60.deb
MD5sum: 63871f6536369f1b084d126ee652ab60
Priority: optional
SHA256: 9f0efcb97653580e12d50447c5a062896a15d3f2eabb6518f218bd5a333baacc
Section: java
Size: 111044248
```

### Steps to Reproduce:

```bash
hal config version edit --version 1.29.2
hal deploy apply
```

### Additional Details:

N/A

**Comment by josh383451** [comment]
Same here. Installing from scratch and getting the same error with 1.29.2. Install works using 1.29.1 though

**Comment by ashleykleynhans** [comment]
Spinnaker 1.29.2 has been deprecated in favor of Spinnaker 1.29.3.
I can confirm that this issue has been resolved in Spinnaker 1.29.3.

**Comment by josh383451** [comment]
Can confirm 1.29.3 is working deploying intended
